### DISCUSSION TITLE
How to disable source maps?

### DISCUSSION THREAD

**Discussion by nicodinh** [zeit-ui/react#751] [Q&A] [answered]
Hi, 

How to disable source maps? 

![image](https://user-images.githubusercontent.com/7486270/156748138-4ae64fa7-5428-4ca9-8fff-f6fe5b819c69.png)

**Comment by unix** [comment]
It depends on what package tool you are using, you can find the option to disable `source-map` in the package tool's documentation.

**Reply by nicodinh** [reply]
@unix thanks for the reply

There are inline sourcemaps in the `dist/` directory after running the commande `yarn run build` from [package.json](https://github.com/geist-org/geist-ui/blob/master/package.json#L25)

I don't know which tool activates them by default, any suggestion?

**Reply by unix** [reply]
This is only the default export of the component library, whether it is used or not depends on the user's scaffolding configuration.
Can you provide a GitHub repo explaining this issue?

**Reply by nicodinh** [reply]
The @geist-ui/core package from the npm registry comes with the inline source maps of all React components.

This example [create-next-app](https://github.com/geist-org/geist-ui/tree/master/examples/create-next-app) is forked from the main repo https://github.com/geist-org/geist-ui/ :

https://github.com/nicodinh/geist-ui-example-create-next-app

`> yarn build && yarn start`

~~https://priceless-brattain-bc65c0.netlify.app/~~

![image](https://user-images.githubusercontent.com/7486270/156846499-3089806e-92f4-4b29-81a1-beac1d829f65.png)

**Comment by unix** [comment, marked as answer]
The output of cjs does produce inline source-map files, and it's indeed hard for the user to remove them.
I created #752  to remove source-map in the cjs bundle, it has been solved in local tests, a version update will be available later.

**Reply by unix** [reply]
BTW: projects that use the esm do not have this issue.

**Reply by unix** [reply]
@nicodinh now you can try the latest version: `yarn add @geist-ui/core@2.3.5`

**Reply by nicodinh** [reply]
Thanks @unix
